stix_shifter_modules/cybereason/requirements.txt

```diff
@@ -0,0 +1 @@
+aiohttp==3.8.3
```

Comment:
This could go in the general requirements file since it's possibly not connector-specific. Also I noticed that `aiohttp-retry` is imported, is that something you could use instead? https://github.com/opencybersecurityalliance/stix-shifter/blob/5.0.1/stix_shifter/requirements.txt#L4

Reply:
I would suggest to update aiohttp-retry to latest version 2.8.3 
https://pypi.org/project/aiohttp-retry/

Reply:
no need for aiohttp==3.8.3 in any requirements file, it is downloaded as a dependancy from [aiohttp-retry](https://github.com/opencybersecurityalliance/stix-shifter/blob/release/5.0.x/stix_shifter/requirements.txt#L4)

Reply:
requirements.txt file is removed